storage/redis/peer_crud_test.go

```diff
@@ -60,8 +62,20 @@ var testPeers = []struct {
 
 func getPeerStore() (*redigomock.Conn, *peerStore) {
 	conn := redigomock.NewConn()
+	pool := redigo.Pool{
+		MaxIdle:     10,
+		IdleTimeout: 240 * time.Second,
+		Dial: func() (redigo.Conn, error) {
+			return conn, nil
```

Comment:
Redigomock does not allow me to have two separate connections as the same mock connection where the expected output has to be registered should be the connection that returns a value. I will remove the idletimeout to fix the potential issue of a connection created earlier timing out.